### ISSUE TITLE
ACE/SERVER: Can't get loot crates out of buildings

### ISSUE THREAD

**Issue by marvis999** [Sparker95/Vindicta#266]
Possible solution:

1. Add drag option to all crates at outposts/police stations
2. set ACE_maxWeightDrag* global variable to ~10000 to allow players to drag them outside

* = might not be exact variable name

**Comment by Sparker95** [comment]
What's the exact variable name? Do you know if they support such a modification? Please research more.

**Comment by marvis999** [comment]
Ok, got it. It was the exact name: **ACE_maxWeightDrag**
Function to make objects draggable is **ace_dragging_fnc_setDraggable**, described here:
https://ace3mod.com/wiki/framework/dragging-framework.html

They do support it. I used it a few times playing on the server, to temporarily fix it. The smaller loot boxes had the drag option already, so I only had to adjust the weight. It now seems like we only have the big green boxes spawning as loot. ACE doesn't even add a drag option to those.
